contracts/src/integrations/cert/router/EigenDACertVerifierRouter.sol

```diff
@@ -21,6 +21,7 @@ contract EigenDACertVerifierRouter is IEigenDACertVerifierRouter, OwnableUpgrade
     error ABNNotGreaterThanLast(uint32 activationBlockNumber);
     error InvalidCertLength();
     error RBNInFuture(uint32 referenceBlockNumber);
+    error LengthMismatch();
```

Comment:
It's not obvious from the name alone what this error means- IMO the error should have a doc describing what this is.

Reply:
https://github.com/Layr-Labs/eigenda/pull/1787/commits/f264b3f04cc48d9fba5bc86d37bdc7d4bd253178